docs/conf.py

```diff
@@ -344,7 +344,7 @@ def configure_doxyfile(
 # show_authors = False
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'sphinx'
+# pygments_style = 'sphinx'
```

Comment:
I think this should not be removed so we can keep the greenish background instead of the gray one in the code snippets.

Reply:
I know but I think gray background color matches better with the rest of the colors in the documentation. @EduPonz  and @MiguelCompany what do you think?

Reply:
I also think the greenish one stands out more than the grey

Reply:
done